src/store/modules/web3.ts

```diff
@@ -95,6 +95,7 @@ const actions = {
       const profiles = await getProfiles([account]);
       commit('WEB3_SET', {
         account,
+        walletConnectType: auth.provider.value?.wc?.peerMeta?.name || 'UnKnown',
```

Comment:
If you are adding a new variable in the state it should be referenced on the state object. Also not fan of this casing "UnKnown" this is a single word, it's better like this:
```suggestion
        walletConnectType: auth.provider.value?.wc?.peerMeta?.name || 'unknown',
```